### PR TITLE
add support for ttlSecondsAfterFinished in fulcio createcerts Job

### PR DESCRIPTION
**Pull request by k4leung4**
Signed-off-by: Kenny Leung <kleung@chainguard.dev>

## Description of the change

add support for ttlSecondsAfterFinished in fulcio createcerts Job

## Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [x] JSON Schema generated.
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/fulcio/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # fulcio
 
+![Version: 0.4.9](https://img.shields.io/badge/Version-0.4.9-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.5.3](https://img.shields.io/badge/AppVersion-0.5.3-informational?style=flat-square)
```
Comment (vaikas) [inline charts/fulcio/README.md:3]: Should this be 0.4.10?

**Review by vaikas — DISMISSED** [review]
Just a question about versioning? Otherwise, LGTM.

File: charts/fulcio/README.md (hunk 1 above)
Comment (k4leung4) [reply charts/fulcio/README.md]: good catch, fixed.

**Review by vaikas — APPROVED** [review]
Thanks!

**Review by hectorj2f — APPROVED** [review]

**Review by cpanato — DISMISSED** [review]

**Review by vaikas — APPROVED** [review]

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**